src/Liquid/AbstractBlock.php

```diff
@@ -129,7 +133,7 @@ protected function whitespaceHandler($token)
 		 * This assumes that TAG_END is always '%}', and a whitespace control indicator
 		 * is exactly one character long, on a third position from the end.
 		 */
-		self::$trimWhitespace = mb_substr($token, -3, 1) === Liquid::get('WHITESPACE_CONTROL');
+		self::$trimWhitespace = $token[-3] === Liquid::$config['WHITESPACE_CONTROL'];
```

Comment:
this requires bumping min php to 7.1 because of negative string offset https://wiki.php.net/rfc/negative-string-offsets

Reply:
We already require 7.4 min, so that's non-issue 🙂